### DISCUSSION TITLE
Grafana.json

### DISCUSSION THREAD

**Discussion by joeflower** [dewenni/ESP_Buderus_KM271#40] [General]
Hallo, erst einmal Danke für dieses wirklich gute Stück Software. Ich habe jedoch Probleme mit der von dir bereitgestellten grafana.json (Vlt. liegt das Problem auch an mir?). Nach dem Einfügen in mein Grafana Umgebung muss ich ja noch die Datenpunkte auswählen. Die Objekte im IOBroker unter Heizung.0.config kann ich in Grafana alle nicht auswählen, obwohl ich sie in Influx speichere. (erscheinen in der Auswahlliste "select measurement" erst gar nicht) Die Objekte unter Heizung.0.status funktionieren jedoch einwandfrei. Eine Recherche im IOBroker Forum brachte mich auch nicht wirklich weiter. 
IOBroker incl. Adapter und Linux sind auf dem neusten Stand und wurden auch schon mehrmals rebootet.

Vlt hat jemand einen Rat? oder gar die gleichen Probleme?

**Comment by dewenni** [comment]
Vielleicht liegt das am Datentyp?  
Die Config Werte sind in der Regel "Strings" und die Status Werte sind "Numbers". So zumindest schreibe ich sie mit Node-Red in die InfluxDB.
Die Config Werte nutze ich in Grafana dann nur in der Tabelle am Ende. Oben das sind alles die Status Werte.
Vielleicht hilft dir diese Info weiter.
